content/blog/2021/02/Kamelet-Catalog/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Kamelet Catalog"
+date: 2021-02-23
+draft: false
+authors: [nicolaferraro]
+categories: ["Camel-K"]
+preview: "We're starting a new initiative at Apache Camel to create a community-driven catalog of reusable Kamelets"
+---
+
+We're starting a new initiative at Apache Camel to create a community-driven catalog of reusable Kamelets (_Camel_ route snipp_ets_, i.e. connectors) that can be used to stream data from/to external systems into any platform powered by Apache Camel.
+
+The ["Apache Camel Kamelet catalog" is available here](/camel-kamelets/latest) and it already contains a collection of useful Kamelets: we would like to extend it with help of the community.
+
+Kamelets are currently supported out-of-the-box by the [Apache Camel K](/camel-k/latest) project and we're working to support them also in Camel core and [Camel Kafka Connector](/camel-kafka-connector/latest/).
```

Comment:
I think we may want to emphaisize a bit more by Camel core it can eventually run anywhere with Camel also Camel Spring Boot and Camel Quarkus.